Extension/src/LanguageServer/configurations.ts

```diff
@@ -301,7 +301,8 @@ export class CppProperties {
             // We don't add system includes to the includePath anymore. The language server has this information.
             let abTestSettings: ABTestSettings = getABTestSettings();
             let rootFolder: string = abTestSettings.UseRecursiveIncludes ? "${workspaceFolder}/**" : "${workspaceFolder}";
-            configuration.includePath = [rootFolder].concat(this.vcpkgIncludes);
+            let defaultFolder: string = "${default}";
+            configuration.includePath = [rootFolder].concat(this.vcpkgIncludes).concat([defaultFolder]);
```

Comment:
I don't think this will work, because it's in if (isUnset(settings.defaultIncludePath))

Reply:
Oh, if C_Cpp.default.includePath equals the default of [], then the includePath is blank (except for compiler includes).

Reply:
I don't think we want to add `"${default}"` if `C_Cpp.default.includePath` is not set.

Reply:
if "C_Cpp.default.includePath" is defined in the settings and it is not null:
     if the c_cpp_properties.json file exists: no changes are required
     if the c_cpp_properties.json file does NOT exist: the ${default} will be added to the includePath.

Reply:
I don't know.